src/mame/konami/konamigv.cpp

```diff
@@ -486,8 +519,19 @@ void tokimeki_state::machine_start()
 	save_item(NAME(m_serial_sensor_data));
 	save_item(NAME(m_heartbeat_signal));
 
+	save_item(NAME(m_printer_bit));
+	save_item(NAME(m_printer_curbit));
+	save_item(NAME(m_printer_curbyte));
+	save_item(NAME(m_printer_is_printing));
+	save_item(NAME(m_printer_current_image));
+	save_item(NAME(m_printer_data));
+	save_item(NAME(m_page_bitmap));
+
 	m_heartbeat_timer = timer_alloc(FUNC(tokimeki_state::heartbeat_timer_tick), this);
 	m_heartbeat_timer->adjust(attotime::zero);
+
+	m_printer_printing_status_timeout = timer_alloc(FUNC(tokimeki_state::printing_status_timeout), this);
+	m_printer_printing_status_timeout->adjust(attotime::never);
```

Comment:
You don’t need to cancel the timer on creating it.  Did you mean to do that on reset?

Reply:
Wouldn't even need it on reset since the timer is only setting m_printer_is_printing to 0 which is already happening in machine_reset. I'll remove it.